Devscord.DiscordFramework/ControllersService.cs

```diff
@@ -67,22 +64,23 @@ private void RunMethods(DiscordRequest request, Contexts contexts, IEnumerable<C
                 {
                     Parallel.ForEach(controllerInfo.Methods, method =>
```

Comment:
przy okazji zmień to na zwykłego foreach
Parallel da więcej problemów niż zysku, bo obsługa wątków wymaga mocy => więcej mocy niż zyskami na współbieżności tutaj